src/engine/blocks.js

```diff
@@ -382,22 +383,38 @@ class Blocks {
                     block.fields[args.name].id = args.value;
                 }
             } else {
+                // Changing the value in a dropdown
                 block.fields[args.name].value = args.value;
+
+                if (!optRuntime || !block.parent){
+                    break;
+                }
+
+                const parent = this._blocks[block.parent];
+                if (parent.isMonitored) {
+                    optRuntime.requestUpdateMonitor(Map({
+                        id: parent.id,
+                        params: this._getBlockParams(parent)
+                    }));
+                }
             }
             break;
         case 'mutation':
             block.mutation = mutationAdapter(args.value);
             break;
-        case 'checkbox':
+        case 'checkbox': {
```

Comment:
What is the curly brace here for?

Reply:
This was required by the lint you mentioned earlier. It ensures that the variable isSpriteSpecific is scoped to just the case, instead of the whole switch statement.